rebasehelper/tests/functional/conftest.py

```diff
@@ -26,12 +26,14 @@
 
 import pytest  # type: ignore
 
-from rebasehelper.constants import RESULTS_DIR, DEBUG_LOG
+from rebasehelper.constants import RESULTS_DIR, DEBUG_LOG, PATCH, REPORT
 
 
 def make_logs_report():
     logs = [
         DEBUG_LOG,
+        PATCH,
+        REPORT + '.json',
```

Comment:
This is a little confusing. Either change the variable from `logs` to `files` for example, or make another list for these two files and iterate through both.

Reply:
And the function name should be changed as well.

Reply:
One list seems like the better way to go. I find the patch and report more important than build logs so I'd prefer to have them closer to the top